packages/openapi-v3/src/__tests__/unit/decorators/response.decorator.unit.ts

```diff
@@ -0,0 +1,204 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/openapi-v3
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Model, model, property} from '@loopback/repository';
+import {expect} from '@loopback/testlab';
+import * as httpStatus from 'http-status';
+import {ResponseObject} from 'openapi3-ts';
+import {get, getControllerSpec, oas} from '../../..';
+
+describe('@oas.response decorator', () => {
+  it('allows a class to not be decorated with @oas.response at all', () => {
+    class MyController {
+      @get('/greet')
+      greet() {
+        return 'Hello world!';
+      }
+    }
+
+    const actualSpec = getControllerSpec(MyController);
+    expect(actualSpec.paths['/greet'].get.responses['200'].description).to.eql(
+      'Return value of MyController.greet',
+    );
+  });
+
+  context('with response models', () => {
+    @model()
+    class SuccessModel extends Model {
+      constructor(err: Partial<SuccessModel>) {
+        super(err);
+      }
+      @property({default: 'ok'})
+      message: string;
+    }
+
+    @model()
+    class FooError extends Model {
+      constructor(err: Partial<FooError>) {
+        super(err);
+      }
+      @property({default: 'foo'})
+      foo: string;
+    }
+
+    @model()
+    class BarError extends Model {
+      constructor(err: Partial<BarError>) {
+        super(err);
+      }
+      @property({default: 'bar'})
+      bar: string;
+    }
+
+    const successSchema: ResponseObject = {
+      description: httpStatus['200'],
```

Comment:
This should require `http-status` to be a dev dependency. The build is passing because the module is transitively added as follows:
```
rest
└─┬ strong-error-handler@3.4.0
  └── http-status@1.4.2 
```